<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poligonos</class>
 <widget class="QDialog" name="Poligonos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pol√≠gonos</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dibujar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXinicio">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x centro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYinicio">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y centro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXfin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lados:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxXinicio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxXfin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="boxYinicio">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
